src/controllers/update/update.manage.trusts.tell.us.about.the.legal.entity.controller.ts

```diff
@@ -85,9 +85,11 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
     if (trust.CORPORATES && trusteeIndex >= 0) {
       const trusteeToChange = trust.CORPORATES[trusteeIndex];
       const updatedTrustee = mapLegalEntityToSession(formData, trusteeToChange);
+      console.log("\n\n\n >>>>>>>>>>>>>>>>>>>>>>>>>>>> WEBAPP update " + JSON.stringify(updatedTrustee) + "\n\n\n");
```

Comment:
console.logs need removing

Reply:
They do but I need them for testing, when I am done they'll be the first to go.